src/kontrol/utils.py

```diff
@@ -112,3 +104,40 @@ def kontrol_file_contents() -> str:
 $ kontrol command --help
 ```
 """
+
+
+def kontrol_toml_file_contents() -> str:
+    return """[build.default]
+foundry-project-root       = '.'
+regen                      = true
+rekompile                  = true
+verbose                    = true
+debug                      = false
+require                    = 'lemmas.k'
+module-import              = 'TestBase:KONTROL-LEMMAS'
```

Comment:
That'll fail on kompilation if `TestBase` is not present, right? Maybe we can use `CounterTest` that get included as part of the Foundry template?

Reply:
`CounterTest` is present only if you run `kontrol init`. If you run `kontrol init --skip-forge` which installs the kontrol dependencies on an already existing foundry project, then it might be missing.
`TestBase` instead is inherited by the forge-std `Test` contract, which all test contracts should inherit.
https://github.com/foundry-rs/forge-std/blob/52715a217dc51d0de15877878ab8213f6cbbbab5/src/Test.sol#L30

Additionally, we could try something like `KontrolCheats`, which is the contract of the Kontrol cheatcodes and is always added with `kontrol init`.

Reply:
Ah right, fair enough, thanks! I forgot `TestBase` is a real contract, I think it's a good choice then. Frankly speaking, because of https://github.com/runtimeverification/kontrol/issues/442, these lemmas are probably applied to all other modules too :).